code/game/verbs/suicide.dm

```diff
@@ -15,7 +15,7 @@
 	var/permitted = 0
 	var/list/allowed = list("Syndicate","traitor","Wizard","Head Revolutionary","Cultist","Changeling")
 	for(var/T in allowed)
-		if(mind.special_role == T)
+		if(mind.special_role == T || ticker.current_state == GAME_STATE_FINISHED)
 			permitted = 1
 			break
```

Comment:
This loop is a bit horrifying. Check if `mind.special_role` is in the allowed list instead while you're here.